scripts/octopus.cake

```
@@ -20,6 +20,10 @@ public class Octo
         var repository = new OctopusAsyncRepository(client);
 
         var proj = await repository.Projects.FindOne(x => x.Name == project);
+        if(proj == null)
+        {
+            context.Error($"Could not find Octopus project {project}");
```

should we throw an exception instead? Like this https://github.com/CKOTech/checkout-ap-template/blob/fc92fa0cf6ef7e25b54c8f27ac6e177cfded0449/scripts/octopus.cake#L25

---

will do